docs/src/appendix/sncast/script/run.md

```diff
@@ -0,0 +1,7 @@
+# `run`
+Compile and run a cairo deployment script.
+
+## `<SCRIPT_NAME>`
+Required.
+
+Script module name that contains the 'main' function that will be executed.
```

Comment:
What about the `package` flag?

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/1320/files#diff-30a17b761a72c6d131dd689b5da672ee47c28ab9e307dd97580ae3443b4113ffL9-L14

Reply:
My bad, thank you for pointing this out.